src/index.html

```diff
@@ -10,10 +10,42 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+      <div class="card-block" data-qa="card">
+        <div class="imac"><img class="block image" src="images/imac.jpeg" alt="card image" /></div>
+        <div class="about-block">
```

Comment:
It's just a text, use `p`
```suggestion
        <p class="about-block">
```